.changeset/red-mails-kick.md

```diff
@@ -0,0 +1,12 @@
+---
+"ember-codemod-remove-global-styles": minor
```

Comment:
I went with a minor release to record the soft-breaking change in indentations in `content-tag`. See https://github.com/ijlee2/codemod-utils/pull/241.